pkg/lib/authenticationflow/declarative/data_password_policy.go

```diff
@@ -10,6 +10,15 @@ type PasswordPolicyHistory struct {
 	Days    int  `json:"days,omitempty"`
 }
 
+type PasswordPolicyExpiry struct {
```

Comment:
It seems to me that there is no need to tell password expiry to the authflow caller. The information we expose in authflow is for the caller to implement the UI. Password expiration details normally will not be shown in the UI. Most of the time, the end-user just enter their password and let the server to tell them whether the password is still valid.

So do not report password expiry in authflow data.